product_pack/readme/USAGE.rst

```diff
@@ -1,10 +1,10 @@
 To use this module, you need to:
 
-#. Go to *Sales > Products > Products*, create a product and set "Is Pack?".
-#. Then set the products has to include in the product pack of the product.
+#. Go to *Sales > Products > Products*, create a product and set *Is Pack?*.
 #. Set *Pack Type* and *Pack component price*.
+#. Then choose the products to include in the pack.
 
-`Product pack` is a base module for `sale_product_pack` and others modules that
+Product pack` is a base module for `sale_product_pack` and other modules that
```

Comment:
Remove both quotes or any, but not half. You can enclose it by double quotes or put its technical name.

Reply:
Done.